diff -u function function.v2
--- function	2025-10-15 16:00:00.000000000 +0200
+++ function.v2	2025-10-15 17:45:00.000000000 +0200
@@ -150,6 +150,8 @@
 # Exécuter une tâche et gérer les erreurs et les logs
 function run_task() {
     local task_key="$1"
     local task_cmd="$2"
     local index="$3"
     local total="$4"
+    echo "$(date '+%F %T'): [DEBUG:run_task] Entrée dans la fonction" >> "${LOGFILE}"
@@ -560,6 +562,9 @@
 # Adresse IP de la machine
 function network_info() {
+    echo "$(date '+%F %T'): [DEBUG:network_info] Entrée dans la fonction" >> "${LOGFILE}"
     INTERFACE=$(ip route get 1.1.1.1 2>/dev/null | awk '/dev/ {print $5; exit}')
     if [[ -z "$INTERFACE" ]]; then
         INTERFACE=$(ip -o link show | awk -F': ' '!/lo|docker|vir|veth/ {print $2; exit}')
@@ -586,6 +591,7 @@
         export IPADRESS
     fi
+    echo "$(date '+%F %T'): [DEBUG:network_info] Fin de la fonction" >> "${LOGFILE}"
 }
@@ -1570,6 +1576,8 @@
 # Mise à jour de GLPI
 function update_glpi() {
+    echo "$(date '+%F %T'): [DEBUG:update_glpi] Entrée dans la fonction" >> "${LOGFILE}"
     info "=== Début de la mise à jour GLPI ===" 1>>"${LOGFILE}"
@@ -1620,6 +1628,7 @@
     info "=== Fin de la mise à jour GLPI ===" 1>>"${LOGFILE}"
     sleep 2
+    echo "$(date '+%F %T'): [DEBUG:update_glpi] Fin de la fonction" >> "${LOGFILE}"
 }
